.github/ISSUE_TEMPLATE/cli_feature.yml

```diff
@@ -8,11 +8,27 @@ body:
   - type: textarea
     id: feature_description
     attributes:
-      label: Please describe your feature request
-      description: Please provide a detailed summary of the feature request including name of buildtest command (buildtest build) and options including intended behavior
+      label: Feature request description
+      description: Please provide a detailed summary of the feature including its intended behavior
     validations:
       required: true
 
+  - type: textarea
+    id: potential_solution
+    attributes:
+      label: Suggest potential solution
+      description: Please describe a solution or action items you would like to propose
+    validations:
+      required: true
+
+  - type: textarea
+    id: notes
+    attributes:
+      label: Additional notes
+      description: Add any other details or notes to consider
```

Comment:
`Add any other details or notes to consider` --> `Please provide any additional information that may include screenshots, attachments`